tsdb/shard.go

```diff
@@ -553,6 +553,9 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]models.Point,
 		tags := p.Tags()
 		if v := tags.Get(timeBytes); v != nil {
 			dropped++
+			if reason == "" {
+				reason = fmt.Sprintf("invalid field name: input tag \"%s\" on measurement \"%s\" is invalid", "time", p.Name())
```

Comment:
Updated both messages; when you accept, I'll update the Godeps in the plutonium PR as this should end up being a fast-forward